docker-compose.yml

```diff
@@ -28,13 +28,16 @@ services:
   reports-consumer:
     image: compliance-backend-rails
     restart: on-failure
+    tty: true
```

Comment:
I found this quite helpful when I was debugging and AFAIK it doesn't cause any problems to include it